Web/Design-Pattern-2/Assignment-3/src/Game.java

```diff
@@ -0,0 +1,49 @@
+public abstract class Game
+{
+	boolean initialize(int noOfTeams, int noOfPlayers) 
```

Comment:
ACTION: Create a method for validate and call that method initialize and initialize method means we are initializing something for class level , but here we are not initializing anything.